components/org.wso2.carbon.identity.api.user.push.device.handler.v1/src/main/resources/push-auth.yaml

```
@@ -0,0 +1,764 @@
+openapi: 3.0.0
+info:
+  version: "v1"
+  title: "WSO2 Identity Server Push-Based Authentication Rest API "
+  description: |
+    This document specifies  **RESTfulAPIs** for WSO2 **Identity Server - Push-based Authentication** .
+    It is written with [openapi:3.0.0 ](http://swagger.io/).
+  contact:
+    name: "WSO2"
+    url: "http://wso2.com/products/identity-server/"
+    email: "architecture@wso2.org"
```

```suggestion
    email: "iam-dev@wso2.org"
```

architecture@wso2.org mail is no longer in use, lets use iam-dev@wso2.org